vms/platformvm/network/network.go

```diff
@@ -122,12 +231,19 @@ func (n *network) AppGossip(ctx context.Context, nodeID ids.NodeID, msgBytes []b
 		return nil
 	}
 	if err := n.issueTx(tx); err == nil {
-		n.gossipTx(ctx, txID, msgBytes)
+		n.legacyGossipTx(ctx, txID, msgBytes)
+
+		n.txPushGossiper.Add(tx)
+		return n.txPushGossiper.Gossip(ctx)
 	}
 	return nil
 }
 
-func (n *network) IssueTx(ctx context.Context, tx *txs.Tx) error {
+// IssueTx verifies the transaction at the currently preferred state, adds
+// it to the mempool, and gossips it to the network.
+//
```

Comment:
nit: this comment seems off

Reply:
I think this comment is mostly accurate, is there something we should change? I just copied this from the prior interface comment that I deleted but I thought this was correct